suite-native/transactions/src/components/TransactionsList/TransactionListItemContainer.tsx

```diff
@@ -172,7 +185,17 @@ export const TransactionListItemContainer = ({
                 )}
                 <Box marginLeft="medium" flex={1}>
                     <HStack flexDirection="row" alignItems="center" spacing="extraSmall">
-                        <Text>{transactionTitle}</Text>
+                        <Box style={applyStyle(titleStyle)}>
+                            <Text variant="body">{transactionTitle}</Text>
+                            {isZeroValuePhishing && (
+                                <Badge
```

Comment:
You're missing a little bit of left padding according to design. Discuss with Lukas if this is fine. If not, just extend with styles prop on the Badge component to add it there.

Reply:
resolved via 9cdf0d6